ui/src/app/customizations/customizations.component.html

```diff
@@ -1,12 +1,9 @@
 <ul class="row nav nav-tabs nav-tabs-pf toolbar-pf">
   <li routerLinkActive="active">
-    <a [routerLink]="['api-connectors']">API Client Connector</a>
+    <a [routerLink]="['api-connectors']">API Client Connectors</a>
```

Comment:
The design reads 'API Client Connector' though: https://github.com/sjcox-rh/syndesis-ux/blob/70f15abbfc56fdd11af34f7adb9cd0ccbda194c9/designs/apiconnector/apiconnector.md

I agree plural sounds better though.

Reply:
Funny on the tech extension design it actually is plural, just figured I'd adjust it accordingly :-)

I think we should start logging bugs for these kinds of things @kahboom WDYT?

Reply:
+1

Reply:
Agreed, let's keep it consistent and make API plural, so API Client Connectors. Sorry for the inconsistency! @kahboom @gashcrumb